app/assets/stylesheets/govuk_publishing_components/components/_heading.scss

```diff
@@ -1,15 +1,10 @@
 .gem-c-heading {
   @include govuk-text-colour;
```

Comment:
Since the text colour is set to with the mixin `govuk-text-colour` mixin on the `govuk-heading-*` classes anyway we could move this to the `.gem-c-heading--font-size-27` to prevent any unnecessary repetition.

Reply:
Great spot, have changed, thanks!